.circleci/config.yml

```
@@ -2,24 +2,48 @@ version: 2
 jobs:
   build:
     docker:
-      - image: 218546966473.dkr.ecr.us-east-1.amazonaws.com/circle-ci:tap-tester
+      - image: 218546966473.dkr.ecr.us-east-1.amazonaws.com/circle-ci:stitch-tap-tester
     steps:
       - checkout
       - run:
           name: 'Setup virtual env'
           command: |
             python3 -mvenv /usr/local/share/virtualenvs/tap-iterable
             source /usr/local/share/virtualenvs/tap-iterable/bin/activate
-            pip install -U 'pip<19.2' setuptools
+            pip install -U 'pip<19.2' 'setuptools<51.0.0'
             pip install .[dev]
+            pip install pylint
       - run:
           name: 'pylint'
           command: |
             source /usr/local/share/virtualenvs/tap-iterable/bin/activate
-            make test
+            pylint tap_iterable -d C,R,W
```

Can't we use standard `$PYLINT_DISABLE_LIST` var here ?

---

If I add `pylint tap_iterable --disable $PYLINT_DISABLE_LIST`, it gives many pylint errors. 
And, formatting changes is not the priority for this beta release.Hence, keeping it as it is.
And, existing pylint issues coming with `pylint tap_iterable -d C,R,W` are fixed in this branch [TDL-22161-fix-date-windowing-logic](https://app.circleci.com/pipelines/github/singer-io/tap-iterable?branch=TDL-22161-fix-date-windowing-logic)

---

```suggestion
            aws s3 cp s3://com-stitchdata-dev-deployment-assets/environments/tap-tester/tap_tester_sandbox dev_env.sh
            source dev_env.sh
            source /usr/local/share/virtualenvs/tap-iterable/bin/activate
            pylint tap_iterable -d "$PYLINT_DISABLE_LIST"
```

It can be done this way, you need to pull env vars from tap_tester_sandbox before using PYLINT_DISABLE_LIST

---

Just letting you know, but this isn't a blocker, u can proceed merging the PR